README.md

```diff
@@ -209,8 +218,41 @@ def MyPhoenixApp.Controller
   end
 end
 ```
+
+`Honeybadger.context/1` stores the context data in the process dictionary, so
+it will be sent with errors/notifications on the same process. The following
+`Honeybadger.notify/1` call will not see the context data set in the previous line.
+
+```elixir
+Honeybadger.context(user_id: 5)
+Task.start(fn ->
+  # this notify does not see the context set earlier
+  # as this runs in a different elixir/erlang process.
+  Honeybadger.notify("Critical error")
```

Comment:
The `notify` function doesn't accept a string argument, this should be something like `%RuntimeError{message: "critical error"}` instead.